src/app.ts

```diff
@@ -1,22 +1,24 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Disabling eslint rules is done in a single file(probably eslintrc. remove it from here), otherwise you will have to put it in every file.